### ISSUE TITLE
issue regarding the use of markdown / html syntax in checkboxes

### ISSUE THREAD

**Issue by espenhalsen** [Altinn/app-frontend-react#1698]
### Description of the bug

When using Markdown or HTML syntax in checkbox text, it displays as plain text instead of converting to links or formatted text. 

This issue occurs with **Altinn.App.Api** and **Altinn.App.Core** versions **7.15.1**, and **Swashbuckle.AspNetCore** version **6.5.0**.

repo: https://altinn.studio/repos/digdir/godkjenn-bruksvilkaar

### Steps To Reproduce

`"<a href" / "[test](test.no)" = plain text`

### Additional Information

![bilde](https://github.com/Altinn/app-frontend-react/assets/89974012/c053dd3f-15cd-4105-b059-15e075ec4063)
![bilde](https://github.com/Altinn/app-frontend-react/assets/89974012/d1cfbcaf-d3e0-4372-a360-654d7d5a0a9a)

**Comment by Febakke** [comment]
We should concider not supporting this feature. It will give us a less accessible solution. 
[MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#interactive_content)

It might also be breaking [3.2.2 on Input](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

**Comment by olemartinorg** [comment]
We don't support this, and probably won't support this (adding links in labels for checkboxes/radios). These are clickable, and adding something else inside them that's clickable might break them. So don't add links here.

In v4+ I think we also remove the link itself and just show the text, not the markdown/html, so this should be a non-issue.
